bootstrap/lib/kernel/ebin/kernel.appup

```diff
@@ -1,7 +1,7 @@
 %% -*- erlang -*-
 %% %CopyrightBegin%
 %%
-%% Copyright Ericsson AB 1999-2022. All Rights Reserved.
+%% Copyright Ericsson AB 1999-2021. All Rights Reserved.
```

Comment:
Nitpick: maybe skip this file?

Reply:
This commit is here for testing only. I intend to remove it before merging.